packages/block-library/src/gallery/editor.scss

```diff
@@ -79,7 +79,7 @@
 			color: rgba(0, 0, 0, 0.2);
 		}
 
-		// Caption placeholder text.
+		// Increase opacity for placeholder text as the gray is lighter.
```

Comment:
I'm not quite sure what this is getting at from just reading the comment. I think it's saying: "Increase opacity for placeholder text on top of images so there's enough contrast whether the image is dark or light."

You could probably reference this PR in the comment to for context:

```scss
// Increase opacity for placeholder text on top of images so there's enough
// contrast whether the image is dark or light.
// See: https://github.com/WordPress/gutenberg/pull/10288
```

🤷‍♂️